src/components/scss/_variables.scss

```diff
@@ -23,7 +23,7 @@ $text-secondary: rgba(0,0,0,0.5);
 // Global breakpoints based on the material-ui default theme
 // "xs": 400,
 // "sm": 600,
-// "md": 960,
+// "md": 1025, // changed from 960
```

Comment:
We should probably stick to the standard breakpoints. Mock ups are mock ups. The mock up isn't responsive so it would be okay to afford some flexibility for the designer. I would not expect the volunteers to have a design for every breakpoint.
We also do not need to comment to let anyone know this was changed form 960 because it will be available in the blame.

Reply:
Let me correct myself. 1024 is a standard breakpoint, 

But the original developer had set these breakpoints up according to [material-ui](https://material-ui.com/customization/breakpoints/) expectations.

I think we'll want to check with design about this, to see what our experience will be like. But if we stick with material-ui breakpoints, I don't expect it being much of an issue.